lib/mix/tasks/firmware.ex

```diff
@@ -198,4 +200,44 @@ defmodule Mix.Tasks.Firmware do
     less_than_elixir_19 = Nerves.elixir_version() |> Version.compare("1.9.0") == :lt
     less_than_elixir_19 && Code.ensure_loaded?(Mix.Tasks.Distillery.Release)
   end
+
+  defp compiler_check() do
+    with {:ok, otpc} <- compiler_version(:code),
+         {:ok, otp_requirement} <- Version.parse_requirement("~> #{otpc.major}.#{otpc.minor}"),
+         {:ok, elixirc} <- compiler_version(:"Elixir.Kernel") do
```

Comment:
`Kernel` should have worked. Elixir should magically expand that to `:"Elixir.Kernel"`.